### PR TITLE
Add dataset-finder link

### PR DESCRIPTION
**Pull request by allandegnan**
### What

https://trello.com/c/ZEJRw8Qy/1181-update-census-hub-page-to-link-to-new-dataset-finder-page

### How to review

DATASET_FINDER_ENABLED=1 ENABLE_GET_DATA_CARD=1 make debug

### Who can review

!me

### CODE REVIEW

File: assets/templates/census-topics.tmpl [hunk 1]
```diff
@@ -91,6 +95,7 @@
                         <p id="other-census-desc">{{ localise "CensusTopicsSubhead" .Language 1 }}</p>
                         <ul class="ons-list ons-list--dashed">
                             <li class="ons-list__item">
+                            fghfgh
```
Comment (lindenmckenzie) [inline assets/templates/census-topics.tmpl:98]: Is there a reason for this? :D

**Review by lindenmckenzie — CHANGES_REQUESTED** [review]
See comment - do you think it's worth adding a default flag setting in config.go?

File: assets/templates/census-topics.tmpl (hunk 1 above)
Comment (allandegnan) [reply assets/templates/census-topics.tmpl]: Manual Debugging (:

**Comment by allandegnan** [comment]
> See comment - also, do you think it's worth adding a default flag setting in config.go?

Added.  

Though false is default, so I probably wouldn't add it except that it's sort of seems the pattern.

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**